### ISSUE TITLE
USER ISSUE: local server disconnection

### ISSUE THREAD

**Issue by MegumiMizu** [StrangeLoopGames/EcoIssues#23379]
\
**User Id: (slg69660)**
**Version:** 0.9.6.0 beta release-334

**Steps to Reproduce:**
join your own local server	

**Expected behavior:**
not being disconected from your own server

**Actual behavior:**
being disconected

**Do you have mods installed? Does the issue happen when no mods are installed?:**
NO

**Please provide log files. Instruction is h
[Player-prev.log](https://github.com/StrangeLoopGames/EcoIssues/files/9465643/Player-prev.log)
[Player.log](https://github.com/StrangeLoopGames/EcoIssues/files/9465644/Player.log)
ere https://github.com/StrangeLoopGames/EcoIssues/wiki/Getting-the-Eco-game-log-file

**Comment by D3nnis3n** [comment]
Please add your server logs: 
https://github.com/StrangeLoopGames/EcoIssues/wiki/Getting-the-Eco-server-and-client-log-files-for-local-world

**Comment by MegumiMizu** [comment]
[log_220831014100.log](https://github.com/StrangeLoopGames/EcoIssues/files/9466347/log_220831014100.log)

**Comment by GeertJanVanDerWolf** [comment]
Happening to me too.

No mod installed. Happens on vanilla game settings.

[2022-09-01.log](https://github.com/StrangeLoopGames/EcoIssues/files/9467549/2022-09-01.log)
[Player-prev.log](https://github.com/StrangeLoopGames/EcoIssues/files/9467552/Player-prev.log)
[Player.log](https://github.com/StrangeLoopGames/EcoIssues/files/9467557/Player.log)

**Comment by D3nnis3n** [comment]
> Happening to me too.
> 
> No mod installed. Happens on vanilla game settings.
> 
> [2022-09-01.log](https://github.com/StrangeLoopGames/EcoIssues/files/9467549/2022-09-01.log) [Player-prev.log](https://github.com/StrangeLoopGames/EcoIssues/files/9467552/Player-prev.log) [Player.log](https://github.com/StrangeLoopGames/EcoIssues/files/9467557/Player.log)

Hey,
please also add your server logs, those only were the client logs. :)

**Comment by MissandeiVR** [comment]
idk what to tell you. i followed the instrctions by the T. D:\SteamLibrary\steamapps\common\Eco\Eco_Data\Server\Logs then i grabbed the log file like you asked for

**Comment by MissandeiVR** [comment]
did you figure out the issue? i like playing online. but also would like to play offline as well.

**Comment by D3nnis3n** [comment]
Have you been disconnected from the internet while trying to start the game? Eco does not support offline play.

So it's quite important to know if you are talking about "singleplayer" or "offline play". The latter is not possible, which is not a bug.

**Comment by MissandeiVR** [comment]
i don't think so. just try to play the local server. on my machine. so single player.
